x-pack/plugins/ml/public/application/routing/routes/jobs_list.tsx

```diff
@@ -32,7 +32,12 @@ export const jobListRouteFactory = (navigateToPath: NavigateToPath, basePath: st
   render: (props, deps) => <PageWrapper {...props} deps={deps} />,
   breadcrumbs: [
     getBreadcrumbWithUrlForApp('ML_BREADCRUMB', navigateToPath, basePath),
-    getBreadcrumbWithUrlForApp('ANOMALY_DETECTION_BREADCRUMB'),
+    getBreadcrumbWithUrlForApp('ANOMALY_DETECTION_BREADCRUMB', navigateToPath, basePath),
```

Comment:
If you are on one of the results views with the flyout open, you can click on the 'Anomaly detection' breadcrumb and the flyout remains open when the Job Management page opens. Can the flyout be automatically closed on moving pages?

![image](https://user-images.githubusercontent.com/7405507/151815126-05ca11e6-5275-43db-b958-4f6d76a5179e.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/124049/commits/a65d457f90b3bca591a38726ca577bdfdf984e81